controllers/glanceapi_controller.go

```diff
@@ -661,7 +522,7 @@ func (r *GlanceAPIReconciler) generateServiceConfigMaps(
 	templateParameters["KeystonePublicURL"] = authURL
 
 	// Select CephBackend (otherwise "file" is the default)
-	gb := glance.SetGlanceBackend(instance)
+	gb := glanceapi.SetGlanceBackend(instance)
 	templateParameters["GlanceBackend"] = gb
 
 	/** If the Glance Backend is Ceph, populate the required templateParameters
```

Comment:
as mentioned at the end, the glance-api.conf is missing a condition for the actual need of the two api instances. default to show_image_direct_url=False and only have the internal one to show_image_direct_url=True. So we should pass the type via the `templateParameters`.